website/content/docs/configuration/replication.mdx

```diff
@@ -0,0 +1,30 @@
+---
+layout: docs
+page_title: Replication - Configuration
+sidebar_title: <code>replication</code>
+description: |-
+  The replication stanza specifies various parameters for tuning replication related values.
+---
+
+# `replication` Stanza
+
+The `replication` stanza specifies various parameters for tuning replication related values.
+
+```hcl
+replication {
+  resolver_discover_servers = true
+  logshipper_buffer_length = 1000
+  logshipper_buffer_size = "5gb"
+}
+```
+
+## `replication` Parameters
+
+- `resolver_discover_servers` `(boolean: true)` - Indicates if the resolver should discover servers.
+- `logshipper_buffer_length` `(integer: 0)` - The maximum number of entries that the log shipper buffer can hold.
+- `logshipper_buffer_size` `(string: "")` - The maximum size that the log shipper buffer can be. This size can be
```

Comment:
Should we add more details about what happens when the value isn't set?

Reply:
Yeah, good call.